source/optional_features/prefill_user_details/index.html.md.erb

```diff
@@ -31,6 +31,13 @@ For example:
 
 All the parameters in `billing_address` are optional.
 
+Each value must be no longer than:
+
+- 254 characters for `email`
+- 255 characters for `cardholder_name`
+- 255 characters for `line1`, `line2` and `city` in `billing_address`
+- 25 characters for `postcode` in `billing_address`
+
 If you include the `country` parameter, it must be an <a href="https://www.iso.org/iso-3166-country-codes.html" target="blank">ISO 3166 code</a>. If the parameter is invalid or missing, the __Country or territory__ field on the payment page will default to 'Great Britain'.
```

Comment:
I know you didn’t change this but since we’re here: it’s specifically ISO 3166-1 alpha-2 codes (we’ve had support queries about this).

Reply:
Good to know, thanks! Added in https://github.com/alphagov/pay-tech-docs/pull/300/commits/d1ea6c6b7e6128ccb832c8fcf6cbf613ca5c09cf.